### DISCUSSION TITLE
How do you emulative the native fzf-vim layout?

### DISCUSSION THREAD

**Discussion by kdog3682** [ibhagwan/fzf-lua#1120] [General]
I have supplied to setup the string: "fzf-native", but the input query is still at the top of the pop-up panel, and not the bottom.

Is there a way to move the input to the bottom of the screen?
Is there a way to truncate files (like native-fzf) whereby /home/kdog will be truncated as ~/ ?

Thank you for the help.

**Comment by ibhagwan** [comment]
> Is there a way to move the input to the bottom of the screen?

```lua
require('fzf-lua').setup(
  fzf_opts = {
    ['--layout'] = 'default',
  }
})
```

> Is there a way to truncate files (like native-fzf) whereby /home/kdog will be truncated as ~/ ?

This should happen automatically, which picker are we talking about `files`, `grep`, `oldfiles`, for example in my `buffers`:
![IMG_2017](https://github.com/ibhagwan/fzf-lua/assets/59988195/598f4fb9-0404-47fe-ae19-b525940542fa)

The reason this might not be happening for native fzf is because there is no processing of the output, whichever command you pass, its output is piped as is to fzf, in that case you can adjust your command args (`fd`, `rg`, etc) by passing a new command with:
```lua
:lua require'fzf-lua'.files({ cmd = … })
```
